### DISCUSSION TITLE
shareinto: Share images (photos, screenshots, etc ...) into Markor

### DISCUSSION THREAD

**Discussion by aisbergde** [gsantner/markor#1606] [Ideas]
#### Description

I would like to share pictures (photos, screenshots, ...) directly to Markor.
Currently this is not possible because Markor is not registered as recipient for pictures.
I need to go to Markor and to find the place where the picture is located I want to append in Markor.

If I could share a picture with Markor it could ask me what to do, for example store a copy into the notes folder.

**Comment by bertisLongfellow** [comment]
Hi. I think the sharing from other apps feature is quite good. I like the ability to share a link in markdown format into an existing document. I then just add an exclamation mark at the beginning, and boom, I have an inline image. Having that as a configurable option (to include a prefixed exclamation mark) would be nice, and I suggest would resolve the original issue, but is not strictly necessary.
